src/main/kotlin/com/hibob/academy/resource/OwnerResource.kt

```diff
@@ -0,0 +1,37 @@
+package com.hibob.academy.resource
+
+import com.hibob.academy.dao.Owner
+import com.hibob.academy.service.OwnerService
+import jakarta.ws.rs.*
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.RequestBody
+
+@Controller
+@Path("/api/noy/owners")
+@Produces(MediaType.APPLICATION_JSON)
+class OwnerResource(private val ownerService: OwnerService) {
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    fun addOwner(@RequestBody owner: Owner): Response {
```

Comment:
do we have to use this annotation @RequestBody ?

Reply:
Because i'm receiving a Json object of the owner